infra/nightly-resources/index.js

```diff
@@ -258,11 +258,12 @@ async function loadBenchmarks(compareTo) {
 
     // Add warnings
     let warningContainer = document.getElementById("warnings");
+    // make warning red
+    warningContainer.style.color = "red";
```

Comment:
Sorry, that's not what I mean. Put a class on the html element in index.html, and add a rule in stylesheet.css. Doing style inline like this is a code smell we should avoid when possible.